wondrous-app/components/Common/NewInviteLinkModal/OrgInviteLink.tsx

```diff
@@ -0,0 +1,219 @@
+import { useEffect, useState } from 'react';
+import * as Sentry from '@sentry/nextjs';
+import {
+  StyledModal,
+  StyledBox,
+  TextHeading,
+  CloseButton,
+  PersonAddIconWrapper,
+  TextHeadingWrapper,
+  HeadingWrapper,
+  IconTextWrapper,
+  InviteButton,
+  InviteThruLinkLabel,
+  InviteThruLinkTextField,
+  InviteThruLinkButtonLabel,
+  InviteThruLinkInputWrapper,
+  StyledDivider,
+  InviteThruEmailLabel,
+  InviteThruEmailTextFieldButtonWrapper,
+  InviteThruEmailTextField,
+  InviteThruLinkSelect,
+  InviteThruLinkMenuItem,
+  InviteThruLinkFormControlSelect,
+  InviteThruEmailTextFieldSelectWrapper,
+  InviteThruEmailButtonLabel,
+  InviteThruLinkButtonSuccessLabel,
+  LinkSwitch,
+  TextSubheading,
+} from './styles';
+import PersonAddIcon from '../../Icons/personAdd';
+import { CopyIcon, CopySuccessIcon } from '../../Icons/copy';
+import { useMutation, useLazyQuery } from '@apollo/client';
+import { CREATE_ORG_INVITE_LINK } from 'graphql/mutations/org';
+import { GET_ORG_ROLES } from 'graphql/queries/org';
+import { useOrgBoard, usePodBoard } from 'utils/hooks';
+import { parseUserPermissionContext } from 'utils/helpers';
+import { LINK, PERMISSIONS } from 'utils/constants';
+
+export const putDefaultRoleOnTop = (roles, permissions) => {
+  if (!roles) return [];
+  roles = [...roles];
+  let defaultRole;
+  let defaultRoleIndex;
+  roles.some((role, index) => {
+    if (role?.default) {
+      defaultRole = { ...role };
+      defaultRoleIndex = index;
+
+      return true;
+    }
+
+    return false;
+  });
+  roles.filter((role) => {
```

Comment:
this filter does nothing :-)

Reply:
this filter is present in `components/Common/InviteLinkModal/OrgInviteLink.tsx`. Shall I remove it from there or let it be ?

Reply:
let's keep it for now